app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -144,12 +144,16 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
   }
 
   /// Queries the task document and updates based on the latest build data.
-  Future<void> updateFirestore(Build build, String sha, String taskName) async {
+  Future<void> updateFirestore(Build build, String commitKeyId, String taskName) async {
     final FirestoreService firestoreService = await config.createFirestoreService();
+    final String sha = commitKeyId.split('/').last;
     final String documentName = '$kDatabase/documents/tasks/${sha}_${taskName}_1';
+    log.info('getting firestore document: $documentName');
     final f.Task firestoreTask =
         await f.Task.fromFirestore(firestoreService: firestoreService, documentName: documentName);
+    log.info('updating firestoreTask based on build');
     firestoreTask.updateFromBuild(build);
+    log.info('finished updating firestoreTask based on builds');
```

Comment:
For these two log messages can we add some kind of identifier? Something like the commit for easier debugging?

Reply:
The `commit` info is included in line 151. These two logs are just used as a way to tell which step in between fails if any. In other words, if we see line 154 or line 156, line 151 should already show up right above.